instrumentation/opentelemetry-instrumentation-pika/src/opentelemetry/instrumentation/pika/utils.py

```diff
@@ -0,0 +1,160 @@
+from typing import Any, Callable, List, Optional
+
+from pika.channel import Channel
+from pika.spec import Basic, BasicProperties
+
+from opentelemetry import context, propagate, trace
+from opentelemetry.instrumentation.utils import _SUPPRESS_INSTRUMENTATION_KEY
+from opentelemetry.propagators.textmap import CarrierT, Getter
+from opentelemetry.semconv.trace import (
+    MessagingOperationValues,
+    SpanAttributes,
+)
+from opentelemetry.trace import Tracer
+from opentelemetry.trace.span import Span
+
+
+class PikaGetter(Getter):  # type: ignore
```

Comment:
I'd suggest to keep everthing but the instrumentor private.

Reply:
I agree. Done